src/main/java/control/DetailViewController.java

```diff
@@ -102,7 +115,8 @@ private void beforePaddingUpdateHelper() {
             DirectorShot directorShot = ((DirectorShot) manager.getActiveShotBlock().getShot());
             
             directorShot.setFrontShotPadding(newVal);
-            directorShot.getCameraShots().forEach(e -> {
+            directorShot.getCameraShots().forEach(
+                e -> {
```

Comment:
You can just leave this one on the line above. This adds an unnecessary line in your method. It does not make the method more readable, it does not make it worse either.